docs/distributions/manjaro/deprecation.md

```diff
@@ -0,0 +1,15 @@
+# Why is Manjaro deprecated?
+
+Read [This.](https://github.com/arindas/manjarno)
+
+TL;DR: It's poorly maintained and dangerous.
```

Comment:
Especially here, I feel like it may be getting a little too opinionated for a wiki ([this](https://en.wikipedia.org/wiki/Wikipedia:Neutral_point_of_view) explains what i mean better than i can).

Maybe saying T2 manjaro will become deprecated and have no maintainer due to these concerns(manjarno link), a good alternative is T2 EndeavourOS would be a bit more neutral?